CoEpi/native/NativeCore.swift

```diff
@@ -41,7 +41,10 @@ protocol TcnGenerator {
 extension NativeCore: ServicesBootstrapper {
 
     // WARN: Must not run long operations. Currently executed synchronously during startup.
-    func bootstrap(dbPath: String) -> Result<(), ServicesError> {        
+    func bootstrap(dbPath: String) -> Result<(), ServicesError> {
+        //CoreLogLevel: 0 -> Trace, ... 4 -> Error; coepi_only logs -> true/false
+        let loggerSetupStatus = setup_logger(CoreLogLevel.init(0), true)
```

Comment:
When I try to run, I get an error on "setup_logger"
am I missing a setup somewhere?

Reply:
@astewart43-gtri Corresponding Rust branch has not been merged/released. To properly run this PR, you need to checkout https://github.com/Co-Epi/app-backend-rust/tree/feature/logging-parametrized and build the Rust lib manually following the instructions on the wiki: https://github.com/Co-Epi/app-backend-rust/wiki/Building-library-for-iOS

Reply:
does this need to wait till the rust branch is merged?